astro/src/content/blog/fusionauth-on-render.mdx

```diff
@@ -0,0 +1,219 @@
+
+Once the logs show `HTTP server listening on port [9011]` you will be ready to access the web service.
+
+### Enter Maintenance Mode
+
+Once the service starts, FusionAuth enters maintenance mode to configure settings:
+
+1. Open the URL provided by Render. It will be slow since this is a very small server.
+2. The setup screen should appear pre-filled based on your environment variables. Click `Submit`.
+
+![Maintenance Mode for FusionAuth](/img/blogs/fusionauth-on-render/5-fusionauth-render.jpg)
+
+### Create Admin User
+
+The next screen that appears will be the FusionAuth Setup Wizard, which will allow you to create the admin user.
+
+Create your admin user in FusionAuth:
+
+- Username: `admin`
+- Email: `admin@example.com`
+- Password: `super_secure_password`
+    
+![FusionAuth Setup Wizard](/img/blogs/fusionauth-on-render/7-fusionauth-render.jpg)
+
+### Access The Admin Dashboard
+
+After setting up the admin user, navigate to the admin dashboard (this should redirect automatically). Here, you might see issues like incorrect requested origins. This is where your proxy setup comes will be needed to address the errors.
+You can read more about why a proxy is needed in the [FusionAuth and Proxies](/docs/operate/deploy/proxy-setup) doc.
+
+## Setting Up a Proxy for FusionAuth
+
+In the below image you can see on the left the proxy version without erros and the non-proxy version with erros on the right.
+
+![FusionAuth with and without Proxy](/img/blogs/fusionauth-on-render/10-fusionauth-render.png)
+
+To resolve configuration issues related to origins and CORS, you will set up a proxy using a simple Node.js application.
+
+1. Clone the [fusionauth-render-proxy](https://github.com/alex-fusionauth/fusionauth-render-proxy.git) repository from GitHub.
+2. Review the code which mainly utilizes Node.js with Express.js.
+    
+```javascript
+const proxy = require("express-http-proxy");
+const app = require("express")();
+
+const port = process.env.PORT || 3001;
+const targetServerURL =
+  process.env.RENDER_FUSIONAUTH_INTERNAL_URL || "localhost:9011";
+
+const renderHostname = process.env.RENDER_EXTERNAL_HOSTNAME;
+
+// Proxy all requests to the fusionauth server
+app.all(
+  "*",
+  proxy(`http://${targetServerURL}`, {
+    proxyReqOptDecorator: (proxyReqOpts, srcReq) => {
+      // You can add custom middleware here if needed
+      if (renderHostname) {
+        proxyReqOpts.headers["X-Forwarded-Host"] = renderHostname;
+        proxyReqOpts.headers["X-Forwarded-Port"] = 443;
+      } else {
+        proxyReqOpts.headers["X-Forwarded-Port"] = port;
+      }
+      return proxyReqOpts;
+    },
+  })
+);
+
+const server = app.listen(port, () =>
+  console.log(`Proxy server listening on port ${port}!`)
+);
+server.keepAliveTimeout = 120 * 1000;
+server.headersTimeout = 120 * 1000;
+```
+3. Deploy this application to Render under a new web service named `FusionAuth Render Proxy`. 
```

Comment:
Should there be steps here to help show how to deploy?

Reply:
It is so straight forward and covered in the video. I really want to push them to the video this is just an incase they need to reference.